### PR TITLE
replace myself with Irene in CODEOWNERS

### PR DESCRIPTION
**Pull request by Eve832**
Modified the CODEOWNER file with Irene as the docs as the codeowner for `/docs/sources/`. Before we merge, I'd like to make sure that Irene has merge rights. Adding Jack B as a reviewer for visibility.

### CODE REVIEW

**Review by irenerl24 — APPROVED** [review]

**Review by jdbaldry — APPROVED** [review]
LGTM, after CI passes and with this approval we can see if Irene is able to merge PRs. 

If Irene doesn't have permission, then we'll need to work with one of the repository admins. Unfortunately my request for owners to step forward in https://raintank-corp.slack.com/archives/C03E2AQKU2W/p1722928970466819 was largely ignored. 

I think our best bet in that case is to ask in #grafana-engineering for someone to give Irene write access.

**Review by tolzhabayev — APPROVED** [review]

**Pull request merged**
